plugins/woocommerce-admin/client/customize-store/transitional/index.tsx

```diff
@@ -90,10 +91,15 @@ export const Transitional = ( {
 					{ __( 'Your store looks great!', 'woocommerce' ) }
 				</h1>
 				<h2 className="woocommerce-customize-store__transitional-subheading">
-					{ __(
-						"Congratulations! You've successfully designed your store. Take a look at your hard work before continuing to set up your store.",
-						'woocommerce'
-					) }
+					{ isEntrepreneurFlow()
+						? __(
+								"Congratuations! You've successfully designed your store. Now you can go back to the Home screen to complete your store setup and start selling.",
+								'woocommerce'
+						  )
```

Comment:
Looking at the translations, they don't exist for this new string yet. I think we could consider using the existing one until the new one is ready.

Reply:
This is fine because Woo 9.0 will only be shipped 3-4 weeks from now. Just need to make it to the code freeze first.

Reply:
> This is fine because Woo 9.0 will only be shipped 3-4 weeks from now. Just need to make it to the code freeze first.

Oh, I see. Makes sense! With knowing that, I think the PR is good to go. 🚀